src/Compilers/Core/Portable/Symbols/IDiscardSymbol.cs

```diff
@@ -14,5 +14,7 @@ public interface IDiscardSymbol : ISymbol
         /// The type of the discarded value.
         /// </summary>
         ITypeSymbol Type { get; }
+
+        Nullability TypeNullability { get; }
```

Comment:
>TypeNullability [](start = 20, length = 15)

Not sure if we need this one. Maybe just for regularity...

Reply:
I debated on this one for a minute as well, but ultimately decided to add it for both the regularity, and because it will likely be needed by QuickInfo as it will still have to show a type for a discard node.

---
In reply to: [205583015](https://github.com/dotnet/roslyn/pull/28860#discussion_r205583015) [](ancestors = 205583015)

Reply:
Drop "Type" from its name.